### DISCUSSION TITLE
how to rewrite response body status by kong?

### DISCUSSION THREAD

**Discussion by oldthreefeng** [Kong/kong#8960] [Help]
Newer to use kong.  
i use apisix before . 

in apisix, i can use respone-rewrite plugin(https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/response-rewrite.md) to implement this .

but in kong,  any plugin here? 

situation.  

`curl a.com/dasd.txt  ` retrun the txt file like "dasd".

**Comment by hanshuebner** [comment]
Hello @oldthreefeng,

thank you for your question.  Have a look at the [request-transformer](https://docs.konghq.com/hub/kong-inc/response-transformer/) plugin which may do what you're looking for.

Kind regards,
Hans
